src/main/resources/application.properties

```
@@ -4,3 +4,7 @@ spring.datasource.password=chocolatefrog
 spring.jpa.hibernate.ddl-auto=update
 spring.liquibase.change-log=classpath:liquibase/changelog-master.yml
 path.to.avatars.folder=/avatars
+logging.level.ru.hogwarts.school.service.StudentService=DEBUG
+logging.level.ru.hogwarts.school.service.FacultyService=DEBUG
+logging.level.ru.hogwarts.school.service.AvatarService=DEBUG
```

Можно просто установить для пакета
```suggestion
logging.level.ru.hogwarts.school.service=DEBUG
```